docs/streams/quickstart.md

```diff
@@ -398,32 +397,111 @@ async function getDCList() {
 <TabItem value="py" label="Python">
 
 ```py
-  from c8 import C8Client
-  import time
-  import base64
-  import six
-  import json
-  import warnings
-  warnings.filterwarnings("ignore")
-
-  print("--- Connecting to C8")
-  # Simple Way
-  client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443,
-                          email='nemo@nautilus.com', password='xxxxx',
-                          geofabric='_system')
-  #--------------------------------------------------------------
-
-  subscriber = client.subscribe(stream="demostream", local=False, subscription_name="test-subscription-1")
-  for i in range(10):
-      print("In ",i)
-      m1 = json.loads(subscriber.recv())  #Listen on stream for any receiving msg's
-      msg1 = base64.b64decode(m1["payload"])
-      print("Received message '{}' id='{}'".format(msg1, m1["messageId"])) #Print the received msg over stream
-      subscriber.send(json.dumps({'messageId': m1['messageId']}))#Acknowledge the received msg.
+subscriber = client.subscribe(stream=demo_stream, local=prefixBool,
+    subscription_name="test-subscription-1")
+for i in range(10):
+    print("In ",i)
+    m1 = json.loads(subscriber.recv())  # Listen on stream for any receiving msg's
+    msg1 = base64.b64decode(m1["payload"])
+    print(F"Received message '{msg1}' id='{m1['messageId']}'") # Print the received msg
+    subscriber.send(json.dumps({'messageId': m1['messageId']})) # Acknowledge the received msg
 ```
 
 </TabItem>
-</Tabs>  
+</Tabs>
+
+
+## Full demo quickstart file
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="Javascript"> 
+
+```js
```

Comment:
Is this supposed to be empty?  Or is it pulling in data from somewhere else?

Reply:
yes. The tutorial is giving the steps in parts. I added this "Full demo quickstart file" section to include the whole file. JS version should be ready after the jsc8 audit is done.

Is it better to remove the JS tab?

Reply:
> yes. The tutorial is giving the steps in parts. I added this "Full demo quickstart file" section to include the whole file. JS version should be ready after the jsc8 audit is done.
> 
> Is it better to remove the JS tab?

Yes, please delete the JS tab until we have a sample ready to provide.  Thank you!

Reply:
OK, done. Thank you @jamesm-macrometa